sdk/python/kfp/components/types/type_utils_test.py

```diff
@@ -79,137 +79,6 @@ def test_is_parameter_type_true(self, type_name, expected_result):
-        },
-        {
-            'artifact_class_or_type_name':
-                artifact_types.ClassificationMetrics,
-            'expected_result':
-                pb.ArtifactTypeSchema(
-                    schema_title='system.ClassificationMetrics',
-                    schema_version='0.0.1')
-        },
-        {
-            'artifact_class_or_type_name':
-                'SlicedClassificationMetrics',
-            'expected_result':
-                pb.ArtifactTypeSchema(
-                    schema_title='system.SlicedClassificationMetrics',
-                    schema_version='0.0.1')
-        },
-        {
-            'artifact_class_or_type_name':
-                artifact_types.SlicedClassificationMetrics,
-            'expected_result':
-                pb.ArtifactTypeSchema(
-                    schema_title='system.SlicedClassificationMetrics',
-                    schema_version='0.0.1')
-        },
-        {
-            'artifact_class_or_type_name':
-                'arbitrary name',
-            'expected_result':
-                pb.ArtifactTypeSchema(
-                    schema_title='system.Artifact', schema_version='0.0.1')
-        },
-        {
-            'artifact_class_or_type_name':
-                _ArbitraryClass,
-            'expected_result':
-                pb.ArtifactTypeSchema(
-                    schema_title='system.Artifact', schema_version='0.0.1')
-        },
-        {
-            'artifact_class_or_type_name':
-                artifact_types.HTML,
-            'expected_result':
-                pb.ArtifactTypeSchema(
-                    schema_title='system.HTML', schema_version='0.0.1')
-        },
-        {
-            'artifact_class_or_type_name':
-                artifact_types.Markdown,
-            'expected_result':
-                pb.ArtifactTypeSchema(
-                    schema_title='system.Markdown', schema_version='0.0.1')
-        },
-        {
-            'artifact_class_or_type_name':
-                'some-google-type',
-            'expected_result':
-                pb.ArtifactTypeSchema(
-                    schema_title='system.Artifact', schema_version='0.0.1')
-        },
-        {
-            'artifact_class_or_type_name':
-                'google.VertexModel',
-            'expected_result':
-                pb.ArtifactTypeSchema(
-                    schema_title='google.VertexModel', schema_version='0.0.1')
-        },
-        {
-            'artifact_class_or_type_name':
-                _VertexDummy,
-            'expected_result':
-                pb.ArtifactTypeSchema(
-                    schema_title='google.VertexDummy', schema_version='0.0.2')
-        },
-    )
-    def test_get_artifact_type_schema(self, artifact_class_or_type_name,
-                                      expected_result):
-        self.assertEqual(
-            expected_result,
-            type_utils.get_artifact_type_schema(artifact_class_or_type_name))
-
```

Comment:
This function no longer needs to handle such a diversity of inputs. It is also tested in a more meaningful way by our compilation tests.